app/components/ui/Footer.jsx

```diff
@@ -4,15 +4,44 @@ import { Link } from 'react-router';
 function Footer() {
   return (
     <footer className="site-footer" role="contentinfo">
-      <div className="footer-content">
-        <div className="footer-text">
-            ©{ new Date().getFullYear() } Shelter Tech
+      <div className="site-footer__content">
+        <div className="site-footer__top">
+          <section className="service-guide">
+            <div className="service-guide__icon" />
+            <h1 className="service-guide__text">
+              SF Service Guide
+            </h1>
+          </section>
+          <section className="site-footer__links">
+            <ul>
+              <h1>About Us</h1>
+              <li><a href="sheltertech.org">Our Story</a></li>
+              <li><a href="sheltertech.org/volunteer">Volunteer</a></li>
+              <li><a href="sheltertech.org/donate">Donate</a></li>
```

Comment:
`/donate` doesn't appear to be a real URL.

Reply:
Ah, I will comment it out.

Reply:
Done